react/PaymentStep.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import { ButtonPlain, IconEdit } from 'vtex.styleguide'
+import { Router } from 'vtex.checkout-container'
+
+import IconMastercard from './icons/IconMastercard'
+import Step from './Step'
+
+const PAYMENT_ROUTE = '/payment'
```

Comment:
Could we face "custom paths" issue here? 🤔 Example:
`https://motorola.com/payment` instead of `https://motorola.com/us/payment`

Reply:
no, we won't because this is relative to the hash route, not the absolute URL

Reply:
Perfect 👍